ProcessMaker/processmaker
# FOUR-6352: Spanish Translation in Field's Validations

# Issue
Ticket: [FOUR-6352](https://processmaker.atlassian.net/browse/FOUR-6352)

For users who are configured to use the Spanish Language, some validation messages are not translated to Spanish:
- There are # validation errors in your form.
- There is a validation error in your form.
- File not allowed.
- File ID does not exist

# Solution
Update or add Spanish translations in Package Translations.

# Steps to Reproduce Issue
1. Create a screen with some required input fields, and a file upload field with a required file type (e.g. pdf)
2. Create a process with a task and assign the screen
3. Log in with a user configured to use the Spanish Language
4. Run a request

# How to Test
1. Update core branch and Package Translations branch to `bugfix/FOUR-6352`.
2. Install Package Translations on core.
3. Repeat Steps to Reproduce Issue.
4. Validation messages should appear in Spanish.

# Code Review Checklist
